cypress/support/commands.js

```diff
@@ -34,3 +34,19 @@ Cypress.Commands.add('getByLabel', (label) => {
             cy.get('#' + id)
         })
 })
+
+Cypress.Commands.add('getByPlaceholder', (placeholder) => {
+    cy.get(`input[placeholder="${placeholder}"]`)
+})
+
+Cypress.Commands.add('login', () => {
+    cy.task('getAllProperties').then(properties => {
+        cy.visitPage('/login');
+        cy.getByLabel('Username').type(properties['login.admin.username'])
+        cy.contains('Continue').click();
+        cy.getByLabel('Password').type(properties['login.admin.password'])
+        cy.contains('Log in').click();
+        cy.contains(properties['login.location']).click();
+        cy.contains('Confirm').click();
+    });
```

Comment:
Let's use the REST API to log in so that we don't have to go through the Login UI for every test. See how the Login microfrontend does it: https://github.com/openmrs/openmrs-esm-core/blob/master/packages/apps/esm-login-app/src/login/login.resource.tsx

Reply:
That's a nice suggestion:

btw, in that case, we should know the UUID of the location, right? Is it okay to hardcode the default location uuid on the properties file? Is it unique for every instance? 

sample implementation:
```
Cypress.Commands.add('loginWithAPI', () => {
    cy.task('getAllProperties').then(properties => {
        const token = window.btoa(`${properties['login.admin.username']}:${properties['login.admin.password']}`);
        cy.request({
            method: 'GET',
            url: `${properties['api.url']}/session`,
            headers: {
                Authorization: `Basic ${token}`,
            },
        })
        cy.request({
            method: 'POST',
            url: `${properties['api.url']}/session`,
            body: { sessionLocation: properties['login.location.uuid'] },
            headers: {
                'Content-Type': 'application/json',
            },
        })
    });
});
```

Reply:
Why the initial GET request?

Yeah we can hardcode the Location UUID here, since we control the backend.

Reply:
Og. got it. We don't need the GET request.